src/_locales/en/messages.json

```diff
@@ -216,27 +216,27 @@
     "version": "Version",
     "welcome_0_close_button": "I'll do it later",
     "welcome_0_restore_key": "I want to restore my backup",
-    "welcome_1a1_didntfind": "Didn't find your account?",
-    "welcome_1a1_title": "Setup which account?",
+    "welcome_1a1_didntfind": "Didn't you find your account?",
+    "welcome_1a1_title": "To set up which account?",
     "welcome_1a1b_connect": "Connect",
     "welcome_1a1b_danger": "Censored network",
     "welcome_1a1b_switch": "Use existing accounts",
     "welcome_1a1b_title": "Maskbook supports these social networks",
-    "welcome_1a2_description": "Then only people you select with Maskbook can see the posted content.",
+    "welcome_1a2_description": "Then, only friends you select with Maskbook can see the posted content.",
     "welcome_1a2_done_button": "Nice!",
-    "welcome_1a2_imgalt": "Screenshot of how to encrypt message with Maskbook",
+    "welcome_1a2_imgalt": "Screenshot of how to encrypt a message with Maskbook",
     "welcome_1a2_title": "Encrypt message in a Maskbook postbox",
     "welcome_1a3_description1": "Our extension is asking you to download your key as a file.",
     "welcome_1a3_description2": "Back it up often.",
     "welcome_1a3_done_button": "Got it!",
     "welcome_1a3_title": "Keep your backups carefully",
-    "welcome_1a3a_connect_subtitle": "will use same crypto key",
+    "welcome_1a3a_connect_subtitle": "will use the same CryptoKey",
     "welcome_1a3a_connect_title": "Connect other accounts",
     "welcome_1a3a_generate_new_key": "Generate",
-    "welcome_1a3a_generate_new_key_password_help": "Please remember the key, or your key will not be able to restore again.",
+    "welcome_1a3a_generate_new_key_password_help": "Please remember the key, or your key will not be able to restore.",
```

Comment:
I suggest "Please remember the key carefully. There is no other way to restore it.".

Reply:
'keep'/'store' -> 'remember' ? @yisiliu